packages/prisma-client/migrations-cli/bin.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+require("ts-node").register({ transpileOnly: true });
```

Comment:
couldn't you use tsup for executing ts? much faster, can't generate typings though, but I am assuming you only need it for execution, check out how its used in sdk/bin for e.g.

Reply:
Ok will try. Took "ts-node" from the examples I'm basing this all on.

Reply:
Do you mean `esbuild-node-loader` like we use it here? https://github.com/webstudio-is/webstudio-designer/blob/7a99e3dfd7d5179e741f8327ddee8c0bde7f86d3/packages/react-sdk/package.json#L10

Reply:
yeah, sorry, tsup is for bundling and the loader is for running, both use esbuild, so its 100x faster than tsc

Reply:
Using `tsx` now